asset/quickstart/ExportAssetsTest/ExportAssetsTest.cs

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GoogleCloudSamples
+{
+    public class ExportAssetsTest
+    {
+        static readonly CommandLineRunner s_runner = new CommandLineRunner
+        {
+            Command = "ExportAssets",
+            VoidMain = ExportAssets.Main,
+        };
+        private readonly ITestOutputHelper _testOutput;
+        private readonly StorageClient _storageClient = StorageClient.Create();
+
+        public ExportAssetsTest(ITestOutputHelper output)
+        {
+            _testOutput = output;
+        }
+
+        [Fact]
+        public void TestExportAsests()
+        {
+            string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+            string bucketName = String.Format("{0}-for-assets", projectId);

```

Comment:
This can still cause problems if two agents are running the test at the same time.  Instead, add a dependency to ../../../../testutil/testutil.csproj and use https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/testutil/TestBucket.cs which will set up and tear down an bucket with a random name.

Reply:
Thanks. Not sure in which situation this can be run concurrently for exactly the same project. If we use random bucket, shall I pass the bucket name to the dev code (../ExportAssets/ExportAssets.cs) with param or with env var?

Reply:
Or, I can use this non-random bucket, but use a random file in dev code, would that sound good for you?

Reply:
Done.